client/components/App.js

```diff
@@ -1,36 +1,55 @@
 
 import React from "react";
 import ProductDetail from './ProductDetail/ProductDetail';
+import RelatedProductsAndOutfits from './RelatedPoductsAndOutfit.jsx'
 import axios from 'axios';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      currentProduct: {}
+      allProducts: [],
```

Comment:
Any difference between allProducts and products?

Reply:
@jincheng411 Heard that with combining 40 and 39. Thanks. And no difference, my mistake. I was writing allProducts in other locations thinking that was state, changed it in state rather than other locations. Selfish change, I can revert. We will talk in 10